test/gtest/ucs/test_pgtable.cc

```diff
@@ -524,7 +524,9 @@ class test_pgtable_perf : public test_pgtable {
 
 private:
     struct region_comparator {
-        bool operator()(ucs_pgt_region_t* region1, ucs_pgt_region_t* region2) {
+        bool
+        operator()(ucs_pgt_region_t *region1, ucs_pgt_region_t *region2) const
```

Comment:
why needed?

Reply:
This change was applied along with cherry pick of [CONFIG/VFS/TEST: Fix compilation on gcc11](https://github.com/openucx/ucx/commit/df43abdd8e4f2773f2f358cd70d111fcc3e05a63)